ee/query-service/app/api/license.go

```diff
@@ -38,3 +72,150 @@ func (ah *APIHandler) applyLicense(w http.ResponseWriter, r *http.Request) {
 
 	ah.Respond(w, license)
 }
+
+func (ah *APIHandler) checkout(w http.ResponseWriter, r *http.Request) {
+
+	type checkoutResponse struct {
```

Comment:
can move this out ?

Reply:
Did you mean out of this function or out of this file to under `/model`? This is a response struct from subscriptions API so I just kept it here since it will not be used anywhere else.

Reply:
Cool, we can move it if in future we use it somewhere. Apart from that things look good to me.